arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -1861,7 +1915,7 @@
         },
         "optimizationType": {
           "description": "Customer can specify what scenario they want this CDN endpoint to optimize, e.g. Download, Media services. With this information we can apply scenario driven optimization.",
-          "type": "string"
+          "$ref": "#/definitions/OptimizationType"
```

Comment:
I'm going to generate the code in the morning to see what exactly the impact will be on the consumer -- but I'm pretty sure that I don't have a problem with this 😄 

I think this is a nice change.